cli/utils.js

```diff
@@ -257,7 +257,25 @@ const getBlueprintPackagePaths = (env, blueprints) => {
     if (!blueprints) {
         return undefined;
     }
-    return blueprints.map(blueprint => {
+
+    const toInstall = Object.entries(blueprints)
```

Comment:
is this the object containing blueprints to install? if yes, can you rename it to `blueprintsToInstall` please